global/scripts/RSNregression.m

```diff
@@ -174,15 +176,21 @@ function RSNregression(InputFile, InputVNFile, GroupMaps, Method, ParamsFile, VA
             [betaICA, NODEts] = weightedDualRegression(normalise(betaICAone), inputConcat, AreaWeights);
         case 'dual'
             [betaICA, NODEts] = weightedDualRegression(GroupMapcii.cdata, inputConcat, AreaWeights);
+        case 'single'
+            SpectraArray = textscan(optional.SpectraParams, '%s', 'Delimiter', {'@'});
+            InputSpectraTS = SpectraArray{1}{1};
+            NODEts=ciftiopen(InputSpectraTS, wbcommand);
+            NODEts=NODEts.cdata';
+            betaICA = ((pinv(normalise(NODEts)) * demean(inputConcat')))';
         otherwise
-            error(['unrecognized method: "' Method '", use "weighted" or "dual"']);
+            error(['unrecognized method: "' Method '", use "weighted," "dual," or "single"']);
```

Comment:
I would put the commas outside the quotes to reduce confusion, this isn't a manuscript.